pages/model/[uuid].tsx

```diff
@@ -168,6 +179,25 @@ function Model() {
     await postEndpoint(`/api/v1/version/${version?._id}/reset-approvals`, {}).then((res) => res.json())
   }
 
+  const handleDelete = () => {
+    setDeleteConfirmContent(<Box />)
+    setDeleteConfirmOpen(true)
+  }
+
+  const handleDeleteCancel = () => {
+    setDeleteConfirmOpen(false)
+  }
+
+  const handleDeleteConfirm = async () => {
+    await deleteEndpoint(`/api/v1/model/${uuid}`, {}).then((res) => {
```

Comment:
We shouldn't be using promise chains if we can avoid it. Instead we can use async/await like so:
```
  const handleDeleteConfirm = async () => {
    const response = await deleteEndpoint(`/api/v1/model/${uuid}`, {})

    if (response.status < 400) {
      router.push('/')
    } else {
      setDeleteConfirmContent(<ConfirmationFeedbackText status={response.status} statusText={response.statusText} />)
    }
  }
```